plaso/output/formatting_helper.py

```diff
@@ -0,0 +1,170 @@
+# -*- coding: utf-8 -*-
+"""Output module field formatting helper."""
+
+from __future__ import unicode_literals
+
+from plaso.lib import errors
+
+
+class FieldFormattingHelper(object):
+  """Output module field formatting helper."""
+
+  # Maps the name of a fields to a a callback function that formats
```

Comment:
Please rewrite this comment to something like: "Maps the name of a field to callback function that formats the field value"

Reply:
done